cli/main.go

```diff
@@ -230,7 +230,15 @@ func main() {
 	}
 	app.ExitErrHandler = func(context *cli.Context, err error) {
 		if err != nil && strings.Contains(err.Error(), clicontext.ErrNoConfig.Error()) {
-			printConfigUsage()
+			if context.Args().Get(context.NArg()-1) == "--help" {
+				err = cli.ShowCommandHelp(context, context.Args().First())
+				if err != nil {
+					cli.HandleExitCoder(err)
+				}
```

Comment:
Minor, but I feel like in this case (calling `--help` and it works) the exit code should be 0 since nothing failed.

Reply:
Fixed!